### ISSUE TITLE
Posenet Models are Encrypted/Corrupted.

### ISSUE THREAD

**Issue by gragundier** [tensorflow/hub#746]
The only models available for pose estimation in tensorflow hub is in js form.  TFJS converter can convert tfjs back to keras models now, but the problem is that the models hosted in tfhub are corrupted.  I know this is true because the model architecture is supposed to look like something like this:
`{"modelTopology":{"class_name":"Sequential","config":{"name":"sequential_3","layers":[{"class_name":"Dense","config":{"units":100,"activation":"relu","use_bias":true,"kernel_initializer":{"class_name":"VarianceScaling","config":{"scale":1,"mode":"fan_in","distribution":"normal","seed":null}},"bias_initializer":{"class_name":"Zeros","config":{}},"kernel_regularizer":null,"bias_regularizer":null,"activity_regularizer":null,"kernel_constraint":null,"bias_constraint":null,"name":"dense_Dense1","trainable":true,"batch_input_shape":[null,14739],"dtype":"float32"}},{"class_name":"Dropout","config":{"rate":0.5,"noise_shape":null,"seed":null,"name":"dropout_Dropout1","trainable":true}},{"class_name":"Dense","config":{"units":2,"activation":"softmax","use_bias":false,"kernel_initializer":{"class_name":"VarianceScaling","config":{"scale":1,"mode":"fan_in","distribution":"normal","seed":null}},"bias_initializer":{"class_name":"Zeros","config":{}},"kernel_regularizer":null,"bias_regularizer":null,"activity_regularizer":null,"kernel_constraint":null,"bias_constraint":null,"name":"dense_Dense2","trainable":true}}]},"keras_version":"tfjs-layers 1.3.1","backend":"tensor_flow.js"},"weightsManifest":[{"paths":["weights.bin"],"weights":[{"name":"dense_Dense1/kernel","shape":[14739,100],"dtype":"float32"},{"name":"dense_Dense1/bias","shape":[100],"dtype":"float32"},{"name":"dense_Dense2/kernel","shape":[100,2],"dtype":"float32"}]}]}`
The model architecture is made human-readable with json.  (Very handy, wish keras saved model had some sort of readme file like this so I don't have to load a model every time just to see its architecture).  However, when you download the models on tfhub.  You get this instead.
`f8b 0808 f065 fa5c 02ff 746d 7069 3134
3830 4300 ed9d 5d6f db36 1486 ff4a e0eb
2611 497d 0ee8 45bb a257 6bb7 76d9 3020
080c 3992 636d 8e65 4872 b2ae e87f 9f24
3b76 1c5b d6e1 c721 6539 eb10 c8b6 4cea
f0f5 ab47 94c8 c3ef 8371 9add 87c5 e0a7
b3c1 5d16 ce27 e7f7 6914 4f07 6fca 97f1
2cce c222 8ede 7fab 3ea5 17d6 8575 1ec5
0fd4 2281 e55a acda e736 9d3d c4d9 7a9f
ab78 96a7 d9c7 69fa 78f1 777e f6f3 eac3
ecec 815c 900b 5a7d a12e fd2a 9da7 d3f4
aefa caf7 c1ac 7ca7 dcb8 2eb7 c2fb 6a6b
902f 46c3 7ae7 745e bdfc 6d1a dec6 9374
1ac5 59f5 6658 1459 fdc5 a8f8 368f ebad
d5c6 e0c3 d5f0 e32f bfbe bb1a fc28 f7cb
27e1 eae3 cd56 94dc 2fab ca93 ffea 6f90
6ad7 cdcb f396 d76c f0e3 e647 f95f f5de
d3d1 7e4a 47c9 b46c aae2 4f72 998e c779
5c0c e965 153a fd30 1c25 61be 09a4 7c33
2fb6 4278 08a7 8b55 0875 cb6d c5b5 09a7
fcca f2f3 df9b 2361 f6f2 d8ca 7d0f 350c
e0d8 1fe3 e46e 521c 3cee 96a6 5718 d60b
015e beb4 a8fd 42a1 5533 34c7 594b 130d
091b ced3 6456 3c26 796c 40ae e591 4b09
b62f 90a3 d66e d326 90a0 a378 5e4c eaa0
d75b 472b e0be 5800 52aa 3c7d ec9c e85a
b452 143b e577 a1c6 1f30 c70f 921e a70b
1d42 c54d 484d 9bb0 3e7a 45bf c3ae 7b70
5729 45a1 13a3 2054 2721 e173 a039 8ba9
0cb9 3706 24a7 6b40 ab27 06b4 4ecf 8056
7f0c 689d ac01 837e f82f 3839 fb05 bd71
5f70 b2e6 f3fb 613e ffe4 cce7 f7c6 7cbe`
For other forms of files, this would probably be acceptable.  However, .json files are supposed to be human readable.  Therefore it is safe to assume that the entire folder is corrupted/encrypted.

**Comment by gragundier** [comment]
Thanks for the fast reply.  As an additional note, this seems to affect all posenet tfjs models.  The hosted tflite model seems to be unaffected.

**Comment by maringeo** [comment]
I forwarded this issue to the authors of the Posenet models. I'll update this issue once there is more information on how the TF.js models were exported.

**Comment by gragundier** [comment]
Great!  As a side note, I found some object detection model that also did key-point recognition, so those worked as a wonderfully performant alternative for the time being.
